Makefile

```diff
@@ -1,19 +0,0 @@
-.PHONY: test install
-
-export FOUNDRY_TEST=test-foundry
-
-install:
-	@git submodule update --init --recursive
-	@yarn
-
-test:
-	@echo Run all tests
-	@forge test -vvv
```

Comment:
Both of those are advertised in the README, we can either keep them or change the doc

Reply:
It would be nice to also add in the doc how to assess gas consumption. There's nothing written written on this for now

Reply:
Do you feel that it is useful to tell people how to run tests when the setup is the one everyone uses ?

Reply:
Same question for you @MerlinEgalite. Everything is written in the foundry book already !

Reply:
We can the test section from the README, or keep part of the makefile. Either is ok for me

Reply:
(there is also the install section)

Reply:
The REAMDE works for me with a mention to the test to check to compare gas is enough

Reply:
>The REAMDE works for me with a mention to the test to check to compare gas is enough

What do you mean ? There are test to compare the gas cost ? with what ?

Reply:
I updated the README

Reply:
I think @QGarchery you were using a randomized test for gas comparison no?

Reply:
Yes `testFullHeapSort` is the best I feel. There is also `testHeapStructure` but a large part of the gas consumption is due to the checking so it's less reliable

Reply:
Ok I see. I dont think that it is worth having this in the README.

Reply:
Ok as you want, it's not clear to me how an outside collaborator will know how to compare gas though